src/webgpu/api/validation/queue/copyImageBitmapToTexture.spec.ts

```diff
@@ -207,7 +227,7 @@ g.test('destination_texture_state')
     t.runTest(
       { imageBitmap },
       { texture: dstTexture },
-      { width: 1, height: 1, depth: 1 },
+      { width: 0, height: 0, depth: 0 },
```

Comment:
Maybe we could have a poptions for this that gives both the noop copy and actual copy cases?